modules/display.js

```diff
@@ -0,0 +1,53 @@
+import Books from './books.js';
+
+const book = JSON.parse(localStorage.getItem('our-books')) || [{}];
+
+const inputTitle = document.querySelector('#text');
+const inputAuthor = document.querySelector('#txt');
+const myButton = document.querySelector('.btn-list');
+const list = document.getElementById('book-list');
+
+export default class UserInterface {
```

Comment:
- [ ] I will advise you to use the ES6 arrow function for the methods in this class. The more the usage, the better in practice. You can see the way I used it in the image below.

![image](https://user-images.githubusercontent.com/86200356/213991972-2ba63208-8738-42ce-924b-bfcd39124b0f.png)